ml-agents/mlagents/trainers/trainer.py

```diff
@@ -170,59 +173,53 @@ def export_model(self) -> None:
         """
         self.policy.export_model()
 
-    def write_summary(self, global_step: int, delta_train_start: float) -> None:
+    def _write_summary(self) -> None:
```

Comment:
Should this just be a stats writer at this point?

Reply:
It's a weird case since stats writers usually write all of the available fields to wherever it writes, but this one only writes reward, time elapsed, and a special bool field that says training/not training.